src/Commands/DecoupledKitCreateCommand.php

```diff
@@ -28,23 +28,21 @@ class DecoupledKitCreateCommand extends CreateCommand implements BuilderAwareInt
      *
      * @param string $site_name Site name
      * @param string $label Site label
+     * @param string $upstream_id Upstream name or UUID
+     *
      * @option org Organization name, label, or ID
      * @option region Specify the service region where the site should be
      *   created. See documentation for valid regions.
      * @option cms Specify the CMS to use for the site.
      *
-     * @usage <site> <label> Creates a new site named <site>, human-readably labeled <label>.
-     * @usage <site> <label> --org=<org> --cms<cms> --install-cms<install-cms> Creates a new site named <site>, human-readably labeled <label>, associated with <organization>, for the specified <cms>.
+     * @usage <site> <label> <upstream> Creates a new site named <site>, human-readably labeled <label>, using code from <upstream>.
+     * @usage <site> <label> <upstream> --org=<org> --cms<cms> --install-cms<install-cms> --region<region> Creates a new site named <site>, human-readably labeled <label>, associated with <organization>, for the specified <cms>.
      *
      * @throws \GuzzleHttp\Exception\GuzzleException
      * @throws \Pantheon\Terminus\Exceptions\TerminusException
      */
-    public function createProject($site_name, $label, $options = ['org' => null, 'region' => null, 'cms' => null, 'install-cms' => TRUE])
+    public function createProject($site_name, $label, $upstream_id, $options = ['org' => null, 'region' => null, 'cms' => null, 'install-cms' => TRUE])
     {
-        $upstreams = [
```

Comment:
Upstream ID should be optional. If it isn't provided, one of these CMS values will be used. If it is provided, the value will be used instead.